### DISCUSSION TITLE
Database Storage Limit Exceeded

### DISCUSSION THREAD

**Discussion by BhaktiDBudar** [supabase/supabase#15662] [Questions]
Supabase provides 500MB Database Storage in free plan. What if this limit is exceeded? What is the cost for extra Storage used?

**Comment by GaryAustin1** [comment]
If you exceed your database will be locked as read only. https://supabase.com/docs/guides/platform/database-size#free-plan-behavior

Pricing page shows PRO plan and cost for storage beyond its higher included limit. https://supabase.com/pricing
